ui/src/app/base/components/NotificationGroupMessage/NotificationGroupMessage.js

```diff
@@ -6,23 +6,28 @@ import { Button } from "@canonical/react-components";
 const NotificationGroupMessage = ({ message, id, action, actionHandler }) => {
   const dispatch = useDispatch();
   return (
-    <p className="p-notification__response u-no-max-width">
-      <span
-        className="p-notification__status"
-        data-test="notification-message"
-        dangerouslySetInnerHTML={{ __html: message }}
-      ></span>
-      {action && (
-        <Button
-          appearance="link"
-          data-test="action-link"
-          className="p-notification__action"
-          onClick={() => actionHandler(id, dispatch)}
-        >
-          {action}
-        </Button>
-      )}
-    </p>
+    <div className="p-notification__response u-no-max-width">
+      <ul className="p-inline-list u-no-margin--bottom">
+        <li className="p-inline-list__item">
+          <span
+            data-test="notification-message"
+            dangerouslySetInnerHTML={{ __html: message }}
```

Comment:
Maybe in future the API could return `message` and `url` so we don't have to do this.